docs/kubernetes/operator/instrumenting-nodejs.md

```diff
@@ -0,0 +1,163 @@
+
+<!--
+Useful links:
+- Example: https://github.com/elastic/elastic-otel-node/tree/main/examples/otel-operator/ documented at https://github.com/elastic/elastic-otel-node/blob/main/DEVELOPMENT.md#testing-k8s-auto-instrumentation-with-otel-operator
+(not user friendly, but we could use it in the future if we want to add a proper example here)
+-->
+
+In this example, you'll learn how to:
+
+- Enable auto-instrumentation of a Node.js application using one of the following supported methods:
+  - Adding an annotation to the deployment Pods.
+  - Adding an annotation to the namespace.
+- Verify that auto-instrumentation libraries are injected and configured correctly.
+- Confirm data is flowing to **Kibana Observability**.
+
+For this example, we assume the application you're instrumenting is a deployment named `nodejs-app` running in the `nodejs-ns` namespace.
+
+1. Ensure you have successfully [installed the OpenTelemetry Operator](./README.md), and confirm that the following `Instrumentation` object exists in the system:
+
+    ```bash
+    $ kubectl get instrumentation -n opentelemetry-operator-system
+    NAME                      AGE    ENDPOINT
+    elastic-instrumentation   107s   http://opentelemetry-kube-stack-daemon-collector.opentelemetry-operator-system.svc.cluster.local:4318
+    ```
+    > [!NOTE]
+    > If your `Instrumentation` object has a different name or is created in a different namespace, you will have to adapt the annotation value in the next step.
+
+2. Enable auto-instrumentation of your Node.js application using one of the following methods:
+
+    - Edit your application workload definition and include the annotation under `spec.template.metadata.annotations`:
+
+    ```yaml
+    kind: Deployment
+    metadata:
+      name: nodejs-app
+      namespace: nodejs-ns
+    spec:
+    ...
+      template:
+        metadata:
+    ...
+          annotations:
+            instrumentation.opentelemetry.io/inject-nodejs: opentelemetry-operator-system/elastic-instrumentation
+    ...
+    ```
+
+    - Alternatively, add the annotation at namespace level to apply auto-instrumentation in all Pods of the namespace:
+
+    ```bash
+    kubectl annotate namespace nodejs-ns instrumentation.opentelemetry.io/inject-nodejs=opentelemetry-operator-system/elastic-instrumentation
+    ```
+
+3. Restart application
+
+    Once the annotation has been set, restart the application to create new Pods and inject the instrumentation libraries:
+
+    ```bash
+    kubectl rollout restart deployment nodejs-app -n nodejs-ns
+    ```
+
+4. Verify the [auto-instrumentation resources](./instrumenting-applications.md#how-auto-instrumentation-works) are injected in the Pods:
+
+    Run a `kubectl describe` of one of your application Pods and check:
+
+**REVIEW NEEDED HERE, it's a copy & paste from java SDK used as javaagent**
+
+    - There should be an init container named `opentelemetry-auto-instrumentation-nodejs` in the Pod:
```

Comment:
Perhaps:

```suggestion
    - There should be an init container named `opentelemetry-auto-instrumentation-nodejs` in the Pod. For exapmle:
```

Though perhaps I'm not giving the reader enough credit to realize that the specific pod name will differ for them.